test/end-to-end/shared/FormUtils.js

```diff
@@ -307,4 +307,12 @@ module.exports = {
 
   // bindings for modal overlay forms
   modal,
+
+  // chains an array of promises and runs them in series.
+  series : async (array, callback) => {
+    return array.reduce(
+      (promise, element, index, array) => promise.then(() => callback(element, index, array)),
+      Promise.resolve(),
```

Comment:
Why do you prefer this model of chaining Promise then just use `Promise.all` ? Does `Promise.all` doesn't order content of the given array each time ?

Reply:
Mainly because Promise.all() doesn't make sense in these cases.  `Promise.all()` runs things in parallel.  But if a user is putting in a list of four accounts to the `bhAccountMultipleSelect` (for example), they do not get four keyboards and start typing all four accounts into the `<input>` at the same time.  Instead, they do the first one, select it, then the next one, then the next one... etc.  That's what `FU.series()` does.

I noticed that `Promise.all()` works locally, but when we went to BrowserStack it crashed each time because it couldn't load the information fast enough.  With FU.series() it gets past that point.